src/app/components/game/interface/turn-overlay/turn-overlay.component.html

```diff
@@ -13,7 +13,7 @@
     <div class="background-line right golden-faded-line"></div>
     <div class="golden-faded-line"></div>
     <div class="own-turn">
-      <div>It´s Your Turn!</div>
+      <div>Du bist drann!</div>
```

Comment:
https://www.duden.de/rechtschreibung/dran

```suggestion
      <div>Du bist dran!</div>
```
Wahlweise auch 

```suggestion
      <div>Du bist am Zug!</div>
```